prime-router/src/test/kotlin/azure/batch/BatchDeciderTests.kt

```diff
@@ -177,4 +182,43 @@ class BatchDeciderTests {
         assertEquals(0, result2.first)
         assertEquals(false, result2.second)
     }
+
+    @Test
+    fun `run should enqueue correct number of batch messages with appropriate delays`() {
+        // ensure this receiver is considered valid and due to batch
+        every { timing1.isValid() } returns true
+        // stub the default‐arg overload for batchInPrevious60Seconds()
+        every { timing1.batchInPrevious60Seconds(any<OffsetDateTime>()) } returns true
+
+        every { timing1.numberPerDay } returns 1
+        every { timing1.maxReportCount } returns 2
+        every { timing1.timeBetweenBatches } returns 120
```

Comment:
Would it be better if instead of mocking this out we set it up in the Receiver to verify it is read properly?